anitya/lib/backends/custom.py

```diff
@@ -28,8 +28,9 @@ class CustomBackend(BaseBackend):
         'http://subsurface.hohndel.org/downloads/',
         'http://www.geany.org/Download/Releases',
     ]
-    more_info = '<br /> More information in the '\
```

Comment:
:-1: for this change this is one of the attribute any backend can use to provide more information